### PR TITLE
Support for debugging nightwatch tests

### PR DESCRIPTION
**Pull request by wtrocki**
Small change for enabling debug mode support.

### CODE REVIEW

**Comment by wtrocki** [comment]
@david-martin What You think about it? Side effect is that change is that we running now nodejs instead of nightwatch command directly.

File: tasks/nightwatchjs.js [hunk 1]
```diff
@@ -10,15 +10,23 @@
...
-  grunt.registerTask('nightwatchjs', 'Grunt plugin for http://nightwatchjs.org/', function(task) {
+  grunt.registerTask('nightwatchjs', 'Grunt plugin for http://nightwatchjs.org/', function (task) {
     var done = this.async();
+    var args;
 
-    var args = ['--env=' + task];
+    if (grunt.option('debug')) {
+      grunt.log.writeln('Running nightwatch in debug mode.');
+      args = ['--debug-brk'];
+    } else {
```
Comment (david-martin) [inline tasks/nightwatchjs.js]: can initialise args to [] above, then no need for this else block

**Comment by david-martin** [comment]
I'm happy to merge this change, with minor tweak above

**Comment by wtrocki** [comment]
@david-martin Changes applied.

**Pull request merged**